en/upgrade/upgrade-from-21-04.md

```diff
@@ -0,0 +1,215 @@
+---
+id: upgrade-from-21-04
+title: Upgrade from Centreon 21.04
+---
+
+This chapter describes how to upgrade your Centreon platform from version 21.04
+to version 21.10.
+
+> If you want to migrate your Centreon server to CentOS / Oracle Linux / RHEL 8
+> you need to follow the [migration procedure](../migrate/migrate-from-20-x.html)
+
+## Perform a backup
+
+Be sure that you have fully backed up your environment for the following
+servers:
+
+- Central server
+- Database server
+
+## Upgrade the Centreon Central server
+
+### Update the Centreon repository
+
+Run the following commands:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--RHEL / CentOS / Oracle Linux 8-->
+```shell
+dnf install -y https://yum.centreon.com/standard/21.10/el8/stable/noarch/RPMS/centreon-release-21.10-1.el8.noarch.rpm
+```
+<!--CentOS 7-->
+```shell
+yum install -y https://yum.centreon.com/standard/21.10/el7/stable/noarch/RPMS/centreon-release-21.10-1.el7.centos.noarch.rpm
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Upgrade PHP
+
+Centreon 21.10 uses PHP in version 8.0.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--RHEL / CentOS / Oracle Linux 8-->
+First, you need to install the **remi** repository:
+```shell
+dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm
+dnf config-manager --set-enabled 'powertools'
+```
+Then, you need to change the PHP stream from version 7.3 to 8.0 by executing the following commands and answering **y**
+to confirm:
+```shell
+dnf module reset php
+dnf module install php:remi-8.0
+```
+<!--CentOS 7-->
+First, you need to install the **remi** repository:
+```shell
+yum install -y https://rpms.remirepo.net/enterprise/remi-release-7.rpm
+```
+Then, you need to enable the php 8.0 repository
+```shell
+yum-config-manager --enable remi-php80
+```
```

Comment:
```suggestion
```shell
yum install yum-utils
yum-config-manager --enable remi-php80
```
```

Reply:
in centos7, yum-utils is not installed by default and should be added to be able to run yum-config-manager (it applies to both update and fresh)